src/main/java/com/users/endpoint/UserEndPoint.java

```diff
@@ -0,0 +1,40 @@
+package com.users.endpoint;
+
+import com.users.dto.UserDto;
+import com.users.entities.User;
+import com.users.service.UserService;
+import lombok.AllArgsConstructor;
+import lombok.extern.slf4j.Slf4j;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.*;
+import java.util.List;
+
+@AllArgsConstructor
+@RestController
+@RequestMapping("/user")
+@Slf4j
+public class UserEndPoint {
+    public final UserService userService;
+
+    @PostMapping
+    public ResponseEntity<UserDto> createStadium(@RequestBody User user) {
+        return ResponseEntity.ok(userService.create(user));
+    }
+
+    @GetMapping("/all-users")
+    public ResponseEntity<List<UserDto>> findAll() {
```

Comment:
adauga log.info la fiecare intrare in aplicatie gen "REST request to find all users" ..

Reply:
face baiatu, il luam impreuna cu debug ul o data ca sa vad mai bine diferentele si de ce se face asa, din ce am citit tine de gradul de severitate si de cat de detaliate sa fie mesajele despre starea aplicatiei.

Reply:
dap, ceva de genu, da nu trb sa le iei cu debug ul ca sa vezi, da o sa ne uitam amandoi